src/js/HScrollMenu.js

```diff
@@ -16,34 +22,71 @@ export default class HScrollMenu extends React.Component {
         this.hiddenNames.push(name)
     }
 
+    pressResetTimeoutButton(event) { 
+        const count = store.getState().ui[store.getState().activeApp].resetTimeout.resetTimeoutValue/1000;
+
+        this.setState({performInteractionCounter: count});
+        UIController.resetPerformInteractionTimeout('UI');      
+    }
+
+    changeCounter() {
+        this.setState(prevState => ({performInteractionCounter: prevState.performInteractionCounter > 0 ? prevState.performInteractionCounter - 1 : ''}))
+    }
+    componentDidMount() {
+        this.interval = setInterval(() => this.changeCounter(), 1000)
+    }
+    componentWillUnmount() {
+        clearInterval(this.interval)
+    }
+
     render() {
         this.clearHiddenNames();
         let menuItems = this.props.data.map((menuItem) => {
             if (menuItem.hidden) {
                 this.pushHiddenName(menuItem.name)
                 return null;
             }
-            return (<div className="hscrollmenu-block"
+
+            return (
+    
+            <div className="hscrollmenu-block"
                 key={menuItem.cmdID + menuItem.name}>
                     <HScrollMenuItem
                         menuItem={menuItem}
                         isPerformingInteraction={this.props.isPerformingInteraction}
                         interactionId={this.props.interactionId ? this.props.interactionId : 0}
                         theme={this.props.theme}
                         onSelection={this.props.onSelection}/>
-                </div>)
+
+            </div>
+            )
         });
 
         var hiddenCommands = null;
         if (this.hiddenNames.length) {
             hiddenCommands = <MenuFooter textAlign="center"/>
         }
 
-        return (
-            <div className="hscrollmenu">
-                { menuItems }
-                { hiddenCommands }
+        let resetTimeoutPInteraction = undefined;
+        if(this.props.interactionId) {
```

Comment:
@LiliiaShlikhtLuxoft What are you trying to check in this condition? `interactionId` is a boolean value? Or  are you checking  existens of `interactionId` ?

Reply:
@ValeriiMalkov yes, I try to check if it's undefined or not. Without this verification do not allow to render the element outside of PerformInteraction call.

Reply:
@LiliiaShlikhtLuxoft If you are trying to check if the property is exist in object please use `in` operator:if( `'interactionId' in this.props`).
If you want to check is it equal with `undefined` please use `if(this.props.interactionId === undefined)`.
It will more readable and prevent possible errors

Reply:
@ValeriiMalkov in this case I check if this variable includes true number (not undefined, not 0).

Reply:
@LiliiaShlikhtLuxoft It is not obviously. Use something like `if(this.props.interactionId !== 0)`

Reply:
Fixed